### DISCUSSION TITLE
General: Pre-release

### DISCUSSION THREAD

**Discussion by infinityofspace** [infinityofspace/jellyfin-alexa-plugin#1] [General]
If you have questions or suggestions about the project that is not enough for an issue, feel free to post them here.

**Please do not ask for an exact release date, because questions about this topic are not answered.**

**Comment by realroywalker** [comment]
Disclaimer: You may be totally un-interested in this, but figured i'd post it on the off chance..  😁 
Unsure if you are familiar (or use) HomeAssistant, but there is currently a package for playing media from multiple sources (of which Jellyfin could be one) on smart speakers.
Unfortunately due to the way Amazon devices work they aren't currently supported as playback targets. This is where your work sprung to my mind, as there is currently discussion around having to create a skill which can then connect to the stream url presented.
I don't know enough about how your current skill works to know if there is any re-use possible, or even if you are interested.. but if so, the discussion link is here; https://github.com/music-assistant/hass-music-assistant/discussions/431

**Reply by infinityofspace** [reply]
Thanks for the comment, I haven't used HA myself yet, but there are some users who have tried to integrate the original Alexa Jellyfin Skill into their system. I don't know if they were successful.

I will have a closer look after the first initial release.
